.pipelines/cg-pipeline.yaml

```diff
@@ -189,7 +189,7 @@ stages:
               targetType: "inline"
               script: |
                 Import-Module -Name "$(Build.SourcesDirectory)\windows\docker\DockerBuildModule.psm1" -Force
-                Build-RetinaAgentImage -fullBuilderImageName $(WINDOWS_BUILDER_IMAGE) -registry $(BUILD_REGISTRY)
+                Build-RetinaAgentImage -fullBuilderImageName $(WINDOWS_BUILDER_IMAGE) -registry $(BUILD_REGISTRY) -appInsightsID $(PROD_AI)
```

Comment:
should this ever be `$TEST_AI`? Use `$ApplicationInsightsID` instead? Based on:
```
if [[ -z $(git status -s) ]]; then
          if [[ -n $(git describe --exact-match --tags HEAD 2>/dev/null) ]]; then
              echo "the current folder is clean and has a tag."
              echo "##vso[task.setvariable variable=ApplicationInsightsID;isOutput=true]$(PROD_AI)"
          else
              echo "the current folder is clean but does not have a tag."
              echo "##vso[task.setvariable variable=ApplicationInsightsID;isOutput=true]$(TEST_AI)"
          fi
      else
          echo "the current folder is not clean."
          echo "##vso[task.setvariable variable=ApplicationInsightsID;isOutput=true]$(TEST_AI)"
      fi
```

Reply:
TEST_AI is currently broken at the moment, so some data is better than no data